cs_upload.py

```diff
@@ -286,51 +334,99 @@ class S3Client:
     object_storage_url: str
     ca_cert: Optional[str]
 
-    def __init__(self, object_storage_url: str, ca_cert: Optional[str]):
+    def __init__(
+        self,
+        object_storage_url: str,
+        ca_cert: Optional[str],
+        check_ca_cert: bool = True,
+        upload_method: str = "POST",
```

Comment:
**suggestion**: Remove the default value for `upload_method` since we always define it in our script (and it doesn't seem to me like `POST` should be the preferred one anyway).